### PR TITLE
Added new streams (Order, Gifts) & new fields in existing streams

### PR DESCRIPTION
**Pull request by cb-senthilvel**

### CODE REVIEW

**Comment by cmerrick** [comment]
Hi @cb-senthilvel, thanks for your contribution!

In order for us to evaluate and accept your PR, we ask that you [sign a contribution license agreement](https://stitch-cla-enforcer.herokuapp.com/). It's all electronic and will take just minutes.

**Comment by dwallace0723** [comment]
@cb-senthilvel I think you will most likely bump up against the same issues we had in #4 where we had to remove the `SELECTED` attribute and add in `REPLICATION_METHOD` and `REPLICATION_KEY` to each Stream to make them compatible with Stitch. I'll let @nick-mckoy confirm. 

If you do make the changes mentioned above, it's probably worth rebasing off of `alpha` at this point so that you can get the other required changes as well (e.g. bumping the `tap-framework` version dependency, and other small tweaks).

**Comment by cb-senthilvel** [comment]
@dwallace0723  removed the SELECTED and added REPLICATION_METHOD, REPLICATION_KEY to each streams.

File: tap_chargebee/schemas/gifts.json [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+					"type": ["null", "string"]
+				},
+				"email": {
+					"type": ["null", "string"]
+				}
+			}
+		},
+		"gift_timelines": {
+			"type": ["null", "array"],
+			"properties": {
```
Comment (KAllan357) [inline tap_chargebee/schemas/gifts.json]: I get an error here as I believe that this `"properties"` key wants to be nested in an `"items"` block.

File: tap_chargebee/schemas/orders.json [hunk 2]
```diff
@@ -0,0 +1,393 @@
+{
+	"type": ["null", "object"],
+	"properties": {
+		"id": {
+			"type": ["null", ""]
```
Comment (KAllan357) [inline tap_chargebee/schemas/orders.json]: I also get an error here as this needs to have a concrete JSON Schema type like `"string"`

**Pull request merged**